docs/hugo/content/guide/best-practices/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Best Practices"
+linkTitle: "Best Practices"
+---
+
+## Managing multiple copies of the same resource
+
+### Transferring resources from one cluster to another
+
+There are two important tenets to remember when transferring resources between clusters:
+1. Don't accidentally delete the resources in Azure during the transfer.
+2. Don't have two instances of ASO fighting to reconcile the same resource to different states.
+
+If you want to migrate all of your ASO resources from cluster A to cluster B, we recommend the following
+pattern:
+
+1. Annotate the resources in cluster A with
+   [serviceoperator.azure.com/reconcile-policy: skip]( {{< relref "annotations#serviceoperatorazurecomreconcile-policy" >}} ).
+   This prevents ASO in that cluster from updating or deleting those resources.
+2. Ensure that cluster B has ASO installed.
+3. `kubectl apply` the resources into cluster B. We strongly recommend an infrastructure-as-code approach where you
+   keep your original/goal-state ASO YAMLs around.
+4. Delete the resources in cluster A. Note that because of the `skip` annotation, this will not delete the backing
+   Azure resources.
```

Comment:
Should include a step to remove annotation for resources in cluster B, once imported?

Reply:
Never actually say to annotate the resources in cluster B, but clarified that in a sub-bullet of step 3.